docs/1-using/1-configuration.mdx

```diff
@@ -23,20 +23,20 @@ Usage:
 
 Flags:
       --api.enabled                                            enable HTTP and gRPC API (default true)
-      --config.path string                                     global conduit configuration file (default "/home/haris/projects/conduitio/conduit/conduit.yaml")
-      --connectors.path string                                 path to standalone connectors' directory (default "/home/haris/projects/conduitio/conduit/connectors")
-      --db.badger.path string                                  path to badger DB (default "/home/haris/projects/conduitio/conduit/conduit.db")
+      --api.grpc.address string                                address for serving the gRPC API (default ":8084")
+      --api.http.address string                                address for serving the HTTP API (default ":8080")
+      --config.path string                                     global conduit configuration file (default "/Users/username/repos/conduit/conduit.yaml")
+      --connectors.path string                                 path to standalone connectors' directory (default "/Users/username/repos/conduit/connectors")
```

Comment:
```suggestion
      --connectors.path string                                 path to standalone connectors' directory (default "/Users/username/repo/conduit/connectors")
```

Reply:
that's my actual directory name lol, but will change it